types/victory/index.d.ts

```diff
@@ -1021,12 +1048,48 @@ declare module "victory" {
      * @default <g/>
      */
     groupComponent?: React.ReactElement;
+    /**
+     * The `minDomain` prop defines a minimum domain value for a chart. This prop is useful in
+     * situations where the minimum domain of a chart is static, while the maximum value depends
+     * on data or other variable information. If the `domain` prop is set in addition to
+     * `minimumDomain`, `domain` will be used.
+     *
+     * Note: The `x` value supplied to the `minDomain` prop refers to the independent variable, and
+     * the `y` value refers to the dependent variable. This may cause confusion in horizontal
+     * charts, as the independent variable will corresponds to the `y` axis.
+     * @example
+     * minDomain={0}
+     * @example
+     * minDomain={{ y: 0 }}
+     */
+    minDomain?: number | Date |  {
```

Comment:
According to [documentation](https://formidable.com/open-source/victory/docs/common-props/#maxdomain) type should not include `Date`

Reply:
Also,  we could create a type for both `maxDomain` and `minDomain` since they both have the same type 😃 
```typescript
type DomainLimit = number | { x?: number; y?: number; };
```

Reply:
Ah, I based it off of this:

https://github.com/FormidableLabs/victory/blob/6a7b68e2f271dd2ff89ae8b6e7e6daa665f6304f/packages/victory-core/src/victory-util/common-props.js#L56-L63

What do you think? Looks like their PropTypes may be wrong. 🤔